oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/importer/IndexImporter.java

```diff
@@ -58,7 +60,7 @@
      */
     static final String ASYNC_LANE_SYNC = "sync";
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(IndexImporter.class);
```

Comment:
Nit: if it's static, then in theory (according to the coding rules), "log" should be "LOG". As you anyway have to change the other constants, I recommend to change this as well.